@rsasaki0511 が話したいトピックを追加する

rsasaki0511 opened the pull request:
本に関しては技術参考書限らず面白い回答が期待できそうかなと思いました。
また、尊敬する人間などいる方であれば考え方が近い人と話ができるきっかけになるのではと思い追加。
最後に技術者になったきっかけの話は自分がよく使うので追加しました！

rsasaki0511 committed:
Update topics.json

june29 commented on docs/topics.json:
```diff
@@ -1,9 +1,12 @@
   "好きなウェブブラウザは？",
-  "最近気になる技術トピックは？"
+  "最近気になる技術トピックは？",
+  "オススメの本は？”,
```
閉じる方のダブルクォートが全角になっている？？？

june29 commented on docs/topics.json:
```diff
@@ -1,9 +1,12 @@
+  "最近気になる技術トピックは？",
+  "オススメの本は？”,
+  "推し技術者は？",
+  "技術者になったきっかけは？",
```
今の実装だと、JSON 配列の最後の要素のうしろに `,` があるとパースできないのでした :pray:
ここは `,` を抜いてください :dash:

june29 commented:
内容はめっちゃいいと思います :relaxed:

rsasaki0511 committed:
Update topics.json

１点追加して指摘事項修正しました。

rsasaki0511 committed:
Update topics.json

褒められた・満足したことと逆に辛かったこと。どちらも記憶に鮮明なことで話やすいかなと思い追加しました。
１０年後～に関しては夢を語る、に近い感じでしょうか。夢を語ってる時って楽しくなることが多いので追加しました。
PCに必ず導入～は語り手が最もPCを活用する理由となるところに繋がると思い追加しました。
好きなガジェットも結構個性的なものから共感できるものに繋がる気がして追加しました。
休日の過ごし方、ちょっとこれは突っ込みすぎかもしれないですが技術力など関係なく誰でも回答できるものでもあり追加しました。
もっとも欲しい物、こちらも技術力など関係なく話しやすいだろうという理由だけで追加しました。

june29 commented on docs/topics.json:
```diff
@@ -1,9 +1,20 @@
 [
   "好きなエディタは？",
-  "好きなプログラミング言語は？",
+  "好きな/興味のあるプログラミング言語は？",
```
なるほど、こっちの方が話しやすいかもですね〜 :bulb:

june29 commented on docs/topics.json:
```diff
@@ -1,9 +1,20 @@
   "好きなウェブブラウザは？",
-  "最近気になる技術トピックは？"
+  "最近気になる技術トピックは？",
+  "オススメの本は？",
```
「本」は技術書に限らず、ジャンルは問わずって感じですかね？

rsasaki0511 replied:
できれば技術書として聞きたいですが、完全に技術者じゃない方もいるみたいなので柔軟な質問にしました。

june29 replied:
よくわかりました :ok_hand:

june29 commented on docs/topics.json:
```diff
@@ -1,9 +1,20 @@
+  "ラッキー！？好きなことについて話す、またはパス",
+  "最も褒められた、満足した仕事は？",
+  "最も苦労した仕事は？",
+  "10年後、どんな人間になっていたい？",
```
他のと比べて、これはなかなか答えるのが大変そう :joy:

june29 reviewed (commented):
ひとつだけ疑問点をコメントしました。そこだけ教えてほしいです！

june29 reviewed (approved):
これでだいぶ充実しますね :heart_eyes:
マージしちゃってください :shipit:
